module/VuFindSearch/src/VuFindSearch/Backend/EDS/Base.php

```diff
@@ -96,6 +96,27 @@ abstract class Base implements LoggerAwareInterface
      */
     protected $searchHttpMethod = 'POST';
 
+    /**
+     * The EDS API Key for this client
+     *
+     * @var ?string
+     */
+    protected $apiKey;
```

Comment:
Should probably explicitly default this to null since there's a chance of it being unset.

```suggestion
    protected $apiKey = null;
```